numba/cuda/cudadrv/nvvm.py

```diff
@@ -343,51 +343,69 @@ def get_log(self):
         return ''
 
 
+COMPUTE_CAPABILITIES = (
+    (3, 5), (3, 7),
+    (5, 0), (5, 2), (5, 3),
+    (6, 0), (6, 1), (6, 2),
+    (7, 0), (7, 2), (7, 5),
+    (8, 0), (8, 6), (8, 7), (8, 9),
+    (9, 0)
+)
+
+# Maps CTK version -> (min supported cc, max supported cc)
```

Comment:
Are the bounds inclusive/exclusive, should this be clarified?

Reply:
Now clarified.